### PR TITLE
Reduced specificity of background color mixin

### PR DESCRIPTION
**Pull request by guywillis**
Resolves https://github.com/adaptlearning/adapt_framework/issues/3161

* Reduces specificity of background color mixin

### CODE REVIEW

**Comment by oliverfoster** [comment]
Is this breaking? @guywillis

**Comment by guywillis** [comment]
It will be breaking if anyone has defined a new block color mixin elsewhere`.block-color-mixin(X, Y);`

**Comment by oliverfoster** [comment]
Can you mitigate this? Or do we need a major version bump?

**Comment by guywillis** [comment]
I could dupe the mixin for the bg implementation and leave the `block-color-mixin` in until a major release to remove it - just a little inefficient but happy to do so

**Review by oliverfoster — APPROVED** [review]

File: less/project/theme-extras.less [hunk 1]
```diff
@@ -192,3 +188,57 @@
...
+// Note: both arguments must be predefined variables
+// --------------------------------------------------
+.bg-color-mixin(black, white);
+.bg-color-mixin(background, background-inverted);
+.bg-color-mixin(transparent-light, black);
+.bg-color-mixin(transparent-dark, white);
+
+.bg-color-mixin(@color, @color-inverted) {
+
+.bg-color.@{color} {
```
Comment (eleanor-heath) [inline less/project/theme-extras.less:208]: indentation looks off here

**Review by eleanor-heath — APPROVED** [review]

File: less/project/theme-extras.less (hunk 1 above)
Comment (guywillis) [reply less/project/theme-extras.less:208]: A little bit of personal preference creeping in to reduce the overall mixin indentation - am happy to indent if you think we should @eleanor-heath

Comment (eleanor-heath) [reply less/project/theme-extras.less:208]: No it's ok to leave it, I would indent it but it is only a very minor and the code is still easy to follow.

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**